tests/qref/data/invalid_yaml_programs.yaml

```diff
@@ -243,4 +243,4 @@
             - N
   description: "Target of a paramater link is not namespaced"
   error_path: "$.program.linked_params[0].targets[0]"
-  error_message: "'N' does not match '^[A-Za-z_][A-Za-z0-9_]*\\\\.[A-Za-z_][A-Za-z0-9_]*'"
+  error_message: "'N' does not match '^[A-Za-z_][A-Za-z0-9_]*\\\\.[A-Za-z_][A-Za-z0-9_]*'"
```

Comment:
**issue:** No newline at the end of the file.

Reply:
Fixed!